README.md

```diff
@@ -6,3 +6,18 @@
 
 
 Python Client for Elemental On-Premises Appliances
+
+## Run Tests
+
+Before running tests locally, install `tox` and `poetry`.
+
+    pip install tox
+    pip install poetry
```

Comment:
I would suggest for us to advocate the use [`pipx`](https://pipxproject.github.io/pipx/) as good practice.